### PR TITLE
View

### PR DESCRIPTION
**Pull request by PromzzyKoncepts**
# Hey There:
> In this PR, I did the project requirements:

- [x] Created html.erb files for each route
- [x] styled the respective routes
- [x] added more data into the database
- [x] ran rubocop

## Kindly Review

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1 +1,21 @@
...
+        <div class= "see-post">
+          <p>Number of posts: <%= user.posts_counter %> </p>
+        
+          <div class= "link">
+            <%= link_to 'Show Post', user %>
+          </div>
+        </div>
+      </div>
+      <%# <div>
+      </div>   %>
```
Comment (Wes-Isaac) [inline app/views/users/index.html.erb]: - [ ] Great job so far, Kindly remove the commented code, commented code makes the code base hard to read.

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,38 @@
...
+      <h2 class='user-name'><%=@user.name%></h2>
+      <p class='post-count'>Number of posts: <%= @user.posts_counter %> </p>
+    </div>
+  </div>
+  <div class= "cards">
+    <%@posts.each do |post| %>
+      <div class= "post-card">
+        <div>
+          <h3>Post #<%= post.id%> </h3>
+          <%# <p><%= post.title%> </p>
```
Comment (Wes-Isaac) [inline app/views/posts/index.html.erb:14]: - [ ] Great job so far, Kindly remove the commented code, commented code makes the code base hard to read.

File: app/assets/stylesheets/user.css [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+.link:hover a {
+  color: antiquewhite;
+  background-color: rgb(150, 52, 52);
+  padding: 10px;
+  border-radius: 8px;
+  transition: all 0.5s ease-out 0.1s;
+}
+
+.user-details {
+  display: flex;
```
Comment (Wes-Isaac) [inline app/assets/stylesheets/user.css:68]: - [ ] Great job so far to be consistent with the wireframe let's add the borders.

### Wireframe
image

### Your Implementation
image

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @PromzzyKoncepts,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Professional README 👍.
- Descriptive PR 💯.
- Use of Gitflow 🥇.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by jerryowusu — APPROVED** [review]
Hi @PromzzyKoncepts , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

CookedDelectableDolphin-size_restricted


## Highlights
No linter errors :heavy_check_mark: 
Great effort so far :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**